cd/application-deployment/perf/perf.env

```diff
@@ -26,6 +26,7 @@ PII_ENABLED=False
 PINPOINT_SMS_VOICE_V2=False
 PLATFORM_STATS_ENABLED=False
 REDIS_ENABLED=True
+REVISED_TEMPLATE_RENDERING=True
```

Comment:
Ditto for perf.

Reply:
I changed it to False (because QA already tested it).